pulsar-client/src/main/java/com/yahoo/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -171,4 +177,46 @@ public ConsumerConfiguration setConsumerName(String consumerName) {
         this.consumerName = consumerName;
         return this;
     }
+
+    /**
+     * @return the maximum amount of messages to be sent in a single RedeliverUnacknowledgedMessages command,
+     * <i>(default: 1000)</i>
+     */
+    public int getMaxRedeliverUnacknowledgedMessagesBatch() {
+        return maxRedeliverUnacknowledgedMessagesBatch;
```

Comment:
@sboobna, @merlimat suggested that we could add an upper bound to the messages sent in a redeliver unacknlowledged messages command, I thought we could make this configurable so that clients have a chance to tweak this.